script/deploy/deploy.ts

```diff
@@ -114,7 +114,10 @@ export const deploy = async () => {
           token,
           siblings,
           hookType,
-          chainAddresses
+          pc[token].mergeInboundWithTokens ?? [],
+          chainAddresses,
+          allAddresses,
+          pc[token]
```

Comment:
since we are passing pc[token] we can avoid passing mergeInboundWithTokens separately.